File: examples/fibonacci.go
package main

import "fmt"

func main() {
	number := 10
	n1, n2 := 0, 1
	var nextTerm int

	fmt.Println("Fibonacci Series in Go:")

	for i := 1; i <= number; i++ {
		fmt.Println(n1)
		nextTerm = n1 + n2
		n1 = n2
		n2 = nextTerm
	}
}

File: examples/fibonacci.ts
const numbers: number = 10;
let num1: number = 0;
let num2: number = 1;
let next: number;

console.log("Fibonacci Series in ts:");

for (let i = 1; i <= numbers; i++) {
  console.log(num1);
  next = num1 + num2;
  num1 = num2;
  num2 = next;
}
